go.mod

```diff
@@ -54,6 +52,8 @@ require (
 	k8s.io/kube-openapi v0.0.0-20240220201932-37d671a357a5 // @grafana/grafana-app-platform-squad
 )
 
+require github.com/grpc-ecosystem/go-grpc-middleware/providers/prometheus v1.0.0
```

Comment:
nit: why not adding this to the group with the other deps from the same source?

Reply:
Don't know just used go get I guess :) feel free to fix together with update to v1.0.1

Reply:
I'll have a look at this